frontend/src/metabase/containers/DashboardPicker.tsx

```diff
@@ -1,7 +1,12 @@
+import type { ComponentProps } from "react";
 import type { CollectionId, DashboardId } from "metabase-types/api";
 import ItemPicker from "./ItemPicker";
 
-export interface DashboardPickerProps {
+export interface DashboardPickerProps
+  extends Pick<
+    ComponentProps<typeof ItemPicker>,
+    "showOnlyPersonalCollections" | "onOpenCollectionChange"
```

Comment:
This change allows TS files that use `DashboardPicker` to pass these 2 new props
1. `showOnlyPersonalCollections`
2. `onOpenCollectionChange`

Reply:
Fixed in https://github.com/metabase/metabase/pull/35137/commits/64bacdcc46001605045bffc7bbde895e13035c76